src/dune_engine/build_system.ml

```diff
@@ -393,7 +392,7 @@ end = struct
           let* action_exec_result =
             let input =
               { Action_exec.root
-              ; context
+              ; context (* can be derived from the root *)
```

Comment:
I'm a bit confused by this comment. Why "can be" if we've already derived it, and why from "root" if we derived it from the `dir`?

Reply:
"root" refers to the field inside `Action_exec.input`

Reply:
I removed the comment though

Reply:
Ah, I see. Thanks!